lib/rouge/tex_theme_renderer.rb

```diff
@@ -113,7 +113,8 @@ def render_blank(tok, &b)
     end
 
     def render_style(tok, style, &b)
-      out = "\\expandafter\\def#{token_name(tok)}#1{"
+      out = String.new
```

Comment:
`String.new` [creates an ASCII-8BIT string](https://ruby-doc.org/core-2.4.0/String.html#method-c-new). To generate UTF-8 String, it ought to be `String.new('')`